samcli/lib/deploy/deployer.py

```diff
@@ -446,7 +446,7 @@ def _check_stack_not_in_progress(status: str) -> bool:
         return "IN_PROGRESS" not in status
 
     def wait_for_execute(
-        self, stack_name: str, stack_operation: str, disable_rollback: bool, execution_time: float = time.time()
+        self, stack_name: str, stack_operation: str, disable_rollback: bool, execution_time: float = time.time() * 1000
```

Comment:
Having anything mutable in the arguments makes me uneasy. In this case though it looks like it should be okay, since its not a list or a dictionary.

Reply:
For execution time parameter where else is this used? If this is the desired value can we just completely deprecate the execution time parameter if we never want values to be passed in?

Reply:
Will take a look at all instances of where this gets called and remove if the references just pass the current time in. If I recall correctly, there was only one instance where it takes in a value other than current time, but even then it is just the time that an execute command finished.

I'll look into it and see if this argument is required.

Reply:
After looking into it, the only time where execution time was provided was when it was called after `execute_changeset()`, however it was just passing the same thing (`time.time()`). This parameter is now removed.